mbf_abstract_nav/src/abstract_controller_execution.cpp

```diff
@@ -185,6 +186,41 @@ void AbstractControllerExecution::setVelocityCmd(const geometry_msgs::TwistStamp
   // TODO so there should be no loss of information in the feedback stream
 }
 
+bool AbstractControllerExecution::checkVelocityDiff(const geometry_msgs::TwistStamped& robot_velocity,
+                                                    const geometry_msgs::TwistStamped& cmd_velocity)
+{
+  // linear total and angular total
+  double robot_linear_total;
+  double robot_angular_total;
+
+  double cmd_linear_total;
+  double cmd_angular_total;
+
+  // compute linear velocity magnitude
+  robot_linear_total = sqrt(pow(robot_velocity.twist.linear.x, 2.0) + pow(robot_velocity.twist.linear.y, 2.0) +
+                            pow(robot_velocity.twist.linear.z, 2.0));
```

Comment:
```suggestion
  robot_linear_total = std::hypot(robot_velocity.twist.linear.x, robot_velocity.twist.linear.y);
```
F: use `std::hypot` for the other places too